frontend/src/app/forms/steps/key-competencies/rating-modal/rating-modal.component.html

```diff
@@ -0,0 +1,79 @@
+<ion-header>
+  <ion-toolbar>
+    <ion-buttons slot="end">
+      <ion-button (click)="closeModal()">Close</ion-button>
+    </ion-buttons>
+  </ion-toolbar>
```

Comment:
Would it here be better to say "Save" or "rate" instead of "Close" ?

Reply:
Currently, when using the slider to update the rating, then the rating is automatically saved. If there is a "Save" button, then as a user, I'd expect that nothing is saved until I press this button. Because of this, I decided to call it "Close". What do you think - is this ok?

In the original mockups, I had such a feature: There was one "Save" and one "Cancel" button. But while implementing this, I figured that having just one "Close" button makes things easier to implement.

Reply:
But when I think of "Close", then I'm not sure if i the data is now saved and transposed. So i thought about something like "Submit" then maybe. But we can ask the team about this, too. :)